content/zh/docs/_index.md

```diff
@@ -1,4 +1,3 @@
 ---
-title: Home
-weight: 5
```

Comment:
why removing this property?

Reply:
this is not needed. thanks.

/lgtm

Reply:
Yes, the weight property does not exist in original en docs. thanks.